septentrion/migration.py

```diff
@@ -60,7 +62,11 @@ def migrate(
                 ):
                     run_script(settings=settings, path=path)
                     logger.info("Saving operation in the database")
+                    # TODO: do this XOR callback ?
+                    # And if callback only, do not create the table ?
```

Comment:
Is it intended as a "WIP" TODO which should be fixed before the merge?

Reply:
no, it's a thought I had while adding the feature that I didn't want to forget as I don't have an answer yet.

Reply:
Ok, maybe it's worth an issue then :) .